src/utils/eth_transaction.cairo

```diff
@@ -245,6 +245,11 @@ namespace EthTransaction {
         let (gas_limit, destination, amount, payload_len, payload, tx_hash, v, r, s) = decode(
             tx_data_len, tx_data
         );
+        if (destination == FALSE){
+            with_attr error_message("ExternallyOwnedAccount: account creations are not payable") {
```

Comment:
I think they are? granted the constructor is payable as well (i think)

Reply:
Just noticed, you're 100% correct.
Fixing that asap.